bin/jsii-release-golang

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+set -eu
+
+###
+#
+# Pushes a directory of golang modules to a GitHub repository.
+#
+# Usage: ./jsii-release-golang [DIR]
+#
+# DIR: directory where the golang modules are located (default is `dist/go`).
+#      Each subdirectory inside it must be a go module and contain a go.mod file.
+#
+# GITHUB_TOKEN       (required): Authentication token that allows to push and tag the repository.
+# VERSION            (required): Module version. Will be used as the repository tag. Required unless NO_TAG is set to a non-zero value.
```

Comment:
This won't work. The version needs to be somehow embedded in the dist and read from there.